arches/app/views/graph.py

```diff
@@ -422,13 +422,20 @@ def post(self, request, graphid=None):
                     clone_data = graph.copy(root=data)
                     clone_data["copy"].slug = None
                     clone_data["copy"].save()
+
                     ret = {"success": True, "graphid": clone_data["copy"].pk}
 
                 elif self.action == "clone_graph":
                     clone_data = graph.copy()
                     ret = clone_data["copy"]
                     ret.slug = None
+                    ret.publication = None
+
                     ret.save()
+
+                    if bool(graph.publication_id):
+                        ret.publish(user=request.user)
```

Comment:
Noticed no tests fail when I revert this. Do you have an idea why that might be?

Reply:
@jacobtylerwalls Just confirming: the new ` test_graph_clone_on_unpublished_graph()` test doesn't fail even when the above change to `arches/app/views/graph.py` is absent?

Reply:
Right. This may be the right changeset after all, but it's just easier for me to verify that as a reviewer if we have a test that fails without the changeset in the application code.

Reply:
@jacobtylerwalls this should be g2g now. It's an interesting interaction with how we're hydrating graphs -- the return of `clone_graph` is correct but instantiating a cloned graph elsewhere in the application will error without the changes.

Reply:
Thanks, that's very helpful context.